<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visitor</class>
 <widget class="QMainWindow" name="visitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="renut">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>170</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>吃喝</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rest">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>170</y>
      <width>171</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>休息</string>
    </property>
   </widget>
   <widget class="QPushButton" name="view">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>161</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>游玩</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>60</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>我的位置（节点编号）：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="my_x">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>60</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>211</width>
      <height>38</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>当前时间;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="time"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sub_pos">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认位置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="sub_time">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>70</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>确认时间</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
